README.nl.md

```diff
@@ -0,0 +1,32 @@
+- [English](README.md)
+- [Dutch](README.nl.md)
+- [日本語](README.ja.md)
+- [한국어](README.ko.md)
+- [Bahasa Indonesia](README.in.md)
+- [Español](README.es.md)
+- [简体中文](README.zh.md)
+
+# Laten-we-gearresteerd-worden project
+
+In Japan werd een schoolmeisje opgepakt door de politie voor het plaatsen van een link naar een website met een eindeloze lustreeks in JavaScript, zoals dit:
+```js
+while (1) {
+  alert("!");
+}
+```
+Gerelateerde nieuwsartikelen:
+
+- (Japans) [https://www3.nhk.or.jp/lnews/kobe/20190304/2020003239.html](https://www3.nhk.or.jp/lnews/kobe/20190304/2020003239.html)
+- (Engels) [https://www.zdnet.com/article/japanese-police-charge-13-year-old-for-sharing-unclosable-popup-prank-online/](https://www.zdnet.com/article/japanese-police-charge-13-year-old-for-sharing-unclosable-popup-prank-online/)
+
+Dus dit wordt beschouwd als een "misdaad" in Japan. Dus laten we criminelen zijn en gearresteerd worden!
+
+Geniet van het criminelen leven!
+
+## Hoe gearresteerd te worden
+
+Het is makkelijk. Fork dit project en vertak het als gh-pagina's. Het is allemaal klaar. Het zou effectiever zijn om de URL "http://youraccount.github.io/lets-get-arrested" op sociale media te delen.
```

Comment:
`Het is makkelijk. Fork dit project en vertak het als gh-pagina's. Het is allemaal klaar. ` ->
`Het is simpel, fork dit project en vertak het als gh-pagina's. Dat is alles.`

`vertak het als gh-pagina's` doesn't sound entirely right still though.